src/server/api/interaction-step.js

```diff
@@ -14,8 +18,18 @@ export const resolvers = {
   InteractionStep: {
     ...mapFieldsToModel([
       'id',
-      'script'
+      'script',
+      'answerOption',
+      'parentInteractionId',
+      'question',
+      'isDeleted'
     ], InteractionStep),
+    questionText: async(interactionStep) => {
```

Comment:
This is to avoid getting the whole question?
Anyway, please refactor to knex query where you can just `.select('question`)` rather than get the whole record.